argo/argo-cluster-generation-from-rancher-capi/01-policy.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+apply:
+- argo-cluster-generation-from-rancher-capi.yaml
```

Comment:
As this is another "generate existing" policy, we should probably also have a step here which creates a Cluster resource first and then installs the policy. The Secret is expected to be generated both after the policy has been installed AND after a second Cluster resource has been created.

Reply:
I also don't see a `00` step to assert the CRD here.

Reply:
I've added.